packages/generators/generators/lib/templates/ts/admin.ts.hbs

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
We should include types for the .ts versions. Do we already have type definitions for the configurations somewhere or would they need to be written?

Reply:
I don't think we have any type definitions here, this snippets are also provided in our documentation 🤷🏼